packages/markdown-docx/src/ToOOXMLVisitor/rules.js

```diff
@@ -169,6 +169,18 @@ const CODE_PROPERTIES_RULE = () => {
     `;
 };
 
+const THEMATICBREAK_RULE = () => {
+    return `
+      <w:p>
+        <w:pPr>
+          <w:pBdr>
+            <w:bottom w:val="single" w:sz="12" w:space="1" w:color="auto"/>
```

Comment:
Do we need the offset (`w:space`) value? [1 point offset is equivalent to 1/72th of an inch](http://officeopenxml.com/WPborders.php) so I don't think it makes much of a difference.

Reply:
@algomaster99 
Does the first one seems fine?
![78f414bf-1e30-4c3e-9309-7cb28989fca9](https://user-images.githubusercontent.com/59891164/125270657-80db0880-e327-11eb-8afd-4e5f2748e9a5.jpeg)

Reply:
The last one is okay. It just needs some offset.